scripts/extensions/sams/src/components/sets/setListItem.tsx

```diff
@@ -78,7 +78,7 @@ export class SetListItem extends React.PureComponent<IProps> {
                     </ListItemRow>
                 </ListItemColumn>
                 <ListItemActionMenu row={true}>
-                    {this.props.deleteSet == null ? null : (
+                    {this.props.deleteSet == null || set.state === SET_STATE.DISABLED && count ? null : (
```

Comment:
I'm not sure checking the state here is required. If the `count > 0` then we should never allow deleting the Set regardless of it's state

Reply:
Resolved